### PR TITLE
Feat/upgrade

### PR DESCRIPTION
**Pull request by Crash--**
commits should be self explanatory but the main objective was to upgrade cozy-client / ui & sharing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,14 +55,14 @@
     "@material-ui/core": "3.9.4",
     "@material-ui/styles": "4.10.0",
     "cozy-bar": "7.7.8",
-    "cozy-client": "16.0.0",
+    "cozy-client": "16.1.1",
```
Comment (ptbrowne) [inline package.json:57]: With gitup "F", it is easy to fixup commits ;)

File: src/targets/browser/index.jsx [hunk 2]
```diff
@@ -128,3 +128,8 @@ export const initApp = () => {
 }
 
 document.addEventListener('DOMContentLoaded', initApp)
+
+if (module.hot) {
+  initApp()
```
Comment (ptbrowne) [inline src/targets/browser/index.jsx:133]: A new client is created, are we sure we want that ?

**Review by ptbrowne — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Crash--) [reply package.json:57]: oublié de squasher... j'ai bien le fixup en plus... Ça m'apprendra à pas relire les commits en soumettant

File: src/components/notes/EditorCorner.spec.jsx [hunk 3]
```diff
@@ -10,13 +13,28 @@ import EditorCorner from './EditorCorner'
 describe('EditorCorner', () => {
   const mockDoc = {}
   it('shows the sharing widget on private views', () => {
-    const readOnlyComponent = shallow(
-      <EditorCorner doc={mockDoc} isPublic={false} isReadOnly={true} />
+    //  useI18n.mockReturnValue({})
```
Comment (ptbrowne) [inline src/components/notes/EditorCorner.spec.jsx:16]: Can you remove the comment ?

File: src/components/notes/EditorCorner.spec.jsx [hunk 4]
```diff
@@ -10,13 +13,28 @@ import EditorCorner from './EditorCorner'
...
+    const readOnlyComponent = mount(
+      <EditorCorner doc={mockDoc} isPublic={false} isReadOnly={true} />,
+      {
+        wrappingComponent: I18n,
+        wrappingComponentProps: {
+          lang: 'en',
+          dictRequire: () => en
+        }
+      }
```
Comment (ptbrowne) [inline src/components/notes/EditorCorner.spec.jsx:25]: Nit: It is useful to make an AppLike component containing all the wrappers necessary for tests.

**Review by ptbrowne — APPROVED** [review]
👍 🎉

File: src/components/notes/EditorCorner.spec.jsx (hunk 4 above)
Comment (Crash--) [reply src/components/notes/EditorCorner.spec.jsx:20]: Thanks!

**Pull request merged**